### DISCUSSION TITLE
Unknown Port 8125

### DISCUSSION THREAD

**Discussion by satya-devloper** [micronaut-projects/micronaut-core#8670] [Q&A]
After updating to micronaut 3.7.5 getting this error

payload.className | io.micrometer.shaded.reactor.netty.channel.FluxReceive
payload.level | WARN
payload.msg | [0094be20, L:/127.0.0.1:40194 - R:localhost/127.0.0.1:8125] An exception has been observed post termination, use DEBUG level to see the full stack: java.net.PortUnreachableException: readAddress(..) failed: Connection refused

Please let me know what I can do to fix this issue

**Comment by shingjo** [comment]
What version of Micronaut were you using previously?

Are you using `io.micronaut.micrometer:micronaut-micrometer-registry-statsd`? Port 8125 is the default port for it.
See: https://micronaut-projects.github.io/micronaut-micrometer/latest/guide/#metricsAndReportersStatsD

**Reply by satya-devloper** [reply]
io.micronaut.micrometer:micronaut-micrometer-registry-statsd, yes using.
Earlier I was using 2.3.3 but now changed to 3.7.5
Then what could be the possible fix to avoid this warning?
